src/index_create_gcloud_aws_iam_roles.js

```diff
@@ -0,0 +1,87 @@
+'use strict';
```

Comment:
Why do we need separate file?

Reply:
for ukgcloud state machine, i have decoupled the two modules (datadog and other iam roles) separately so that we can invoke each part and automate it if required in future.  Thats why i have created two separate files.